examples/eco/05-offshore-h2/eco-h2-offshore.py

```diff
@@ -17,240 +10,21 @@
-
-    print("wind capacity factor: ", np.sum(hopp_results["combined_pv_wind_power_production_hopp"])*1E-3/(plant_config["plant"]["capacity"]*365*24))
-
-    if use_profast:
-        lcoe, pf_lcoe = he_fin.run_profast_lcoe(plant_config, orbit_project, capex_breakdown, opex_breakdown_annual, hopp_results, design_scenario, verbose=verbose, show_plots=show_plots, save_plots=save_plots)
-        lcoh_grid_only, pf_grid_only = he_fin.run_profast_grid_only(plant_config, orbit_project, electrolyzer_physics_results, capex_breakdown, opex_breakdown_annual, hopp_results, design_scenario, total_accessory_power_renewable_kw, total_accessory_power_grid_kw, verbose=verbose, show_plots=show_plots, save_plots=save_plots)
-        lcoh, pf_lcoh = he_fin.run_profast_full_plant_model(plant_config, orbit_project, electrolyzer_physics_results, capex_breakdown, opex_breakdown_annual, hopp_results, incentive_option, design_scenario, total_accessory_power_renewable_kw, total_accessory_power_grid_kw, verbose=verbose, show_plots=show_plots, save_plots=save_plots)
-
-    ################# end OSW intermediate calculations
-    power_breakdown = he_util.post_process_simulation(lcoe, lcoh, pf_lcoh, pf_lcoe, hopp_results, electrolyzer_physics_results, plant_config, h2_storage_results, capex_breakdown, opex_breakdown_annual, orbit_project, platform_results, desal_results, design_scenario, plant_design_scenario, incentive_option, solver_results=solver_results, show_plots=show_plots, save_plots=save_plots)#, lcoe, lcoh, lcoh_with_grid, lcoh_grid_only)
-
-    # return
-    if output_level == 0:
-        return 0
-    elif output_level == 1:
-        return lcoh
-    elif output_level == 2:
-        return lcoh, lcoe, capex_breakdown, opex_breakdown_annual, pf_lcoh, electrolyzer_physics_results
-    elif output_level == 3:
-        return lcoh, lcoe, capex_breakdown, opex_breakdown_annual, pf_lcoh, electrolyzer_physics_results, pf_lcoe, power_breakdown
-
-# run the stuff
-if __name__ == "__main__":
-
-    # storage_types = ["pressure_vessel", "turbine", "pressure_vessel", "pressure_vessel", "pressure_vessel", "pressure_vessel"]
-    # scenarios = [0,3,4,5,6,7]
-    # for scenario, storage_type in zip(scenarios, storage_types): # range(7): # [3,]:
-    #     run_simulation(verbose=False, show_plots=False, save_plots=True,  use_profast=True, incentive_option=1, plant_design_scenario=scenario, storage_type=storage_type)
-
-    ## this should result in  5.13 $/kg LCOH
-    run_simulation(verbose=True, show_plots=True, save_plots=False,  use_profast=True, incentive_option=1, plant_design_scenario=0)
+    
+    run_simulation(filename_turbine_config, filename_orbit_config, filename_floris_config, verbose=False, show_plots=False, save_plots=True,  use_profast=True, incentive_option=1, plant_design_scenario=1)
```

Comment:
Needs to be updated to load in filename_hopp_config and filename_eco_config. Will need to provide files in input folder.

Reply:
This has been done.